juju/juju
# Move agent/logger facade to model config service.

The logger agent facades use model config for the `logging-config` key, which is used to configure the loggers on each agent. This work changes the logger facade to use the model config service.

## QA steps

- Unit tests.
- Bootstrap
- Debug-log and change the logging-config for the model to trace and see trace logs show up.
- Test for k8s and iaas models, with machines and units correctly updating their logging config.

## Documentation changes

N/A

## Links

**Jira card:** JUJU-
